src/sortStudents.ts

```diff
@@ -1,7 +1,74 @@
-// describe Student type
-// create and export SortType enum
-// create SortOrder type
+export interface Student {
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
+}
+
+export enum SortType {
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
+}
+
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder = 'asc',
+): Student[] {
+  const copy = [...students].map((student) => ({
+    ...student,
+  }));
```

Comment:
```suggestion
  const copy = [...students];
```
Would the code work the same if you simplify this, or am I missing something?

Reply:
no, you are not missing anything. i have no idea why i created new objects. strange stuff on my side and nice catch on yours